charts/jenkins/Chart.yaml

```diff
@@ -37,7 +37,7 @@ annotations:
     - name: jenkins
       image: jenkins/jenkins:2.426.3-jdk17
     - name: k8s-sidecar
-      image: kiwigrid/k8s-sidecar:1.24.4
```

Comment:
should we prefix this with docker.io so we only get one PR ref https://github.com/jenkinsci/helm-charts/pull/991

@kvanzuijlen

Reply:
Ah, yes, looks like that's required. I'll open a PR shortly prefixing every image with docker.io